### ISSUE TITLE
Cannot get your instructions to work.

### ISSUE THREAD

**Issue by rygo6** [yegor256/s3auth#268]
I'm fairly experienced with all the subject matters necessary to make this work, but following your instructions I simply cannot get this to work. It just asks for the password over and over again.

I have gotten everything set up to the point where going to the htpasswd file does return bytes.
So I presume my DNS routing is correct and that I have the correct key in place to get into my s3 bucket. I have tried generating my .htpasswd file multiple ways, with MD5 and SHA, from command line and the referred website in the instructions. Doesn't work. I have had a colleague step through the instructions one step at a time with me to ensure I didn't do anything wrong. We are now both out of ideas on what could be wrong.

The one area of slight discrepancy it seems to me is:
{
  "Statement": [
    {
      "Effect": "Allow",
      "Action": ["s3:GetObject", "s3:GetBucketWebsite"],
      "Resource": [
        "arn:aws:s3:::bucket-1.example.com/*",
        "arn:aws:s3:::bucket-2.example.com/*"
      ]
    }
  ]
}

What exactly goes in place of "bucket-1.example.com"? I have tried putting my DNS host name. The bucket name. The url to the static host website of the bucket.

**Comment by brynbstn7** [comment]
rygo6 - - I am having the same issue - - how did you solve this ?

**Comment by yegor256** [comment]
@rygo6 you should put the name of the bucket there. and don't forget to give our servers 10 minutes to update their memory caches

**Comment by brynbstn7** [comment]
I've tried 3 cases:
CNAME recordset name:     arn:aws:s3:::mytest.example.com/*
Bucket name:     arn:aws:s3:::bucket-01/*
Bucket + domain:      arn:aws:s3:::bucket-01.example.com/*

...waiting 30 minutes for update cache ... and get the repeating login prompt issue ....  which is supposed to work?

**Comment by yegor256** [comment]
Should be like this: `arn:aws:s3:::bucket-01/*`

Make sure you configured the domain and bucket name correctly in the account page at s3auth.com. What did you enter there?

**Comment by brynbstn7** [comment]
I entered the bucket name in the bucket name field:  "bucket-01" for example.
I have multiple registered domains created.  One is the root domain:  metasources.org.  And another is the name of the CNAME record step:  testauth.metasources.org.  The 3rd is completely irrelevant.  I tried deleting it, but guess that doesn't work.   Also the user name, to which I attached the above policy, is the same name as is listed in my .htpasswd file.  And the CNAME record step points to relay.s3auth.com and not an alias is checked.  For the S3 bucket, it's a public website, for permissions it has a record for "Authenticated Users", with List selected.  I cannot grant permission to this user specifically - - it is not an option.   

Not sure what went wrong in this setup....
